rust/operator-binary/src/extensions.rs

```diff
@@ -38,5 +38,18 @@ pub fn get_extension_list(
         extensions.push(EXT_S3.to_string());
     }
 
+    // Add user specified extensions to the list of loaded extensions if any are present
+    for additional_extension in &druid.spec.cluster_config.additional_extensions {
+        if extensions.contains(additional_extension) {
```

Comment:
O(n^2) here we come :) Just kidding, you could use a BTreeMap - feel free to ignore